swf/src/read.rs

```diff
@@ -28,6 +29,28 @@ pub fn parse_swf(swf_buf: &SwfBuf) -> Result<Swf<'_>> {
     })
 }
 
+/// Extracts an SWF inside of an SWZ file.
+pub fn extract_swz(input: &[u8]) -> Result<Vec<u8>> {
```

Comment:
Any reason not introducing this function directly in `loader.rs`, since it's used only there?

Reply:
Well I felt like SWZ was really just a wrapper for an SWF file, so it should be included as part of the SWF library. I can make it separate, but it feels weird to me